templates/portfolios/portfolios_table.html

```diff
@@ -29,15 +29,13 @@
     {% if "can_grant_portfolio_owner_permission" in perms %}
     <div class="col-md-2">Owner</div>
         <div class="col-md-2">
-            <span class="glyphicon glyphicon-pencil"></span>
-            <a id="new-project" class="portfolio-project-link" onclick="show_manage_portfolio_modal('{{portfolio.id}}', '{{ portfolio.title }}'); return false;" href="#">Manage Portfolio</a>
+            <a id="" class="portfolio-project-link" href="/portfolios/{{ portfolio.id }}/edit" title="Edit portfolio"><span class="glyphicon glyphicon-pencil"></span></a>
         </div>
     {% elif "change_portfolio" in perms%}
     <div class="col-md-2">Portfolio Member</div>
     <div class="col-md-2">
         <div class="col-md-2">
-            <span class="glyphicon glyphicon-pencil"></span>
-            <a id="new-project" class="portfolio-project-link" onclick="show_manage_portfolio_modal('{{portfolio.id}}', '{{ portfolio.title }}'); return false;" href="#">Manage Portfolio</a>
+            <a id="" class="portfolio-project-link" href="/portfolios/{{ portfolio.id }}/edit" title="Edit portfolio"><span class="glyphicon glyphicon-pencil"></span></a>
         </div>
```

Comment:
I'm not following the logic here for this if-else condition. If the link is the same either way, is the if-else needed? If it's just a matter of printing "Owner" or "Portfolio Member", maybe the scope of the if-else statement should be limited to that line.
Also, with the text of the link removed (i.e. "Manage Portfolio") in favor of the edit pencil icon, you may need to add some help text or aria labels for screen readers to help impaired users identify what this icon is for.

Reply:
An Owner has all the permissions of a portfolio member and can grant portfolio owner permission.

Reply:
I guess what I'm saying here is that they both have access to the same link (the "a" tag is basically duplicated between the 'if' and 'else' statement), so maybe that part should be pulled out of the 'if' statement and just have the "Owner"/"Portfolio Manager" text in the 'if' statement (i.e. the part that is different).